### PR TITLE
CR-1155592: Removed unnecessary debug message

### PR DESCRIPTION
**Pull request by sawanm-xilinx**
Fix: Removed unnecessary cout statements.

### CODE REVIEW

File: src/runtime_src/core/pcie/emulation/hw_emu/alveo_shim/shim.cxx [hunk 1]
```diff
@@ -1482,7 +1482,7 @@ namespace xclhwemhal2 {
 
     if (xclemulation::config::getInstance()->isFastNocDDRAccessEnabled())
     {
-      std::cout << "Checking Write Fastmem " << dest << std::endl;
+      // std::cout << "Checking Write Fastmem " << dest << std::endl;
       if (mNocFastAccess.isAddressMapped(dest, size))
       {
         std::cout << "Writing Fastmem " << dest << std::endl;
```
Comment (venkatp-xilinx) [inline src/runtime_src/core/pcie/emulation/hw_emu/alveo_shim/shim.cxx:1488]: Is there any reason why we did not remove this std::cout? Please remove even this

File: src/runtime_src/core/pcie/emulation/hw_emu/alveo_shim/shim.cxx [hunk 2]
```diff
@@ -1562,7 +1562,7 @@ namespace xclhwemhal2 {
     //Need to be removed when NOC allows transaport_dbg access
     if (xclemulation::config::getInstance()->isFastNocDDRAccessEnabled())
     {
-      std::cout << "Checking Read Fastmem " << src << std::endl;
+      // std::cout << "Checking Read Fastmem " << src << std::endl;
       if (mNocFastAccess.isAddressMapped(src, size))
       {
         mNocFastAccess.read(src, (unsigned char *)dest, size);
```
Comment (venkatp-xilinx) [inline src/runtime_src/core/pcie/emulation/hw_emu/alveo_shim/shim.cxx:1568]: Is there any reason why we did not remove this std::cout? Please remove even this

File: src/runtime_src/core/pcie/emulation/hw_emu/alveo_shim/shim.cxx (hunk 1 above)
Comment (sawanm-xilinx) [reply src/runtime_src/core/pcie/emulation/hw_emu/alveo_shim/shim.cxx]: Removed.

File: src/runtime_src/core/pcie/emulation/hw_emu/alveo_shim/shim.cxx (hunk 2 above)
Comment (sawanm-xilinx) [reply src/runtime_src/core/pcie/emulation/hw_emu/alveo_shim/shim.cxx:1567]: Removed.

Comment (venkatp-xilinx) [reply src/runtime_src/core/pcie/emulation/hw_emu/alveo_shim/shim.cxx:1567]: I see its commented. I think its better to remove

**Review by venkatp-xilinx — APPROVED** [review]

File: src/runtime_src/core/pcie/emulation/hw_emu/alveo_shim/shim.cxx (hunk 2 above)
Comment (sawanm-xilinx) [reply src/runtime_src/core/pcie/emulation/hw_emu/alveo_shim/shim.cxx:1565]: Done, removed all the 4 lines having unnecessary cout.

**Review by chvamshi-xilinx — APPROVED** [review]

**Comment by chvamshi-xilinx** [comment]
ok to test

**Comment by chvamshi-xilinx** [comment]
retest this please

**Comment by dayeh-xilinx** [comment]
retest this please.

**Pull request merged**